packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -201,6 +209,39 @@ function Preview( { item, categoryId, viewType } ) {
 	);
 }
 
+function Author( { item, viewType } ) {
+	const [ isImageLoaded, setIsImageLoaded ] = useState( false );
+	const { text, icon, imageUrl } = useAddedBy( item.type, item.id );
+	const withIcon = viewType !== LAYOUT_LIST;
+
+	return (
+		<HStack alignment="left" spacing={ 1 }>
+			{ withIcon && imageUrl && (
+				<div
+					className={ classnames(
+						'page-templates-author-field__avatar',
```

Comment:
This style rule is defined in `packages/edit-site/src/components/page-templates-template-parts/style.scss` and is bundled as part of edit site stylesheet. It's the same as the existing Preview field below (`page-templates-preview-field`).